cocotb/decorators.py

```diff
@@ -360,6 +392,9 @@ def __get__(self, obj, type=None):
             and standalone functions"""
         return self.__class__(self._func.__get__(obj, type))
 
+    def __str__(self):
+        return str(self._func.__name__)
```

Comment:
If we need this at all, can you instead use
```
def __repr__(self):
    return "external({!r})".format(self._func)
```

Reply:
Honestly, the `__str__` methods are mostly useless. All of these classes and the regression manager have a massive and inconsistent public interface. If I can't break the public interface, I was hoping for consistency at a minimum. I'd prefer to remove it.